_sass/members.scss

```diff
@@ -0,0 +1,228 @@
+background-color: #ffffff;
```

Comment:
probably a bit of nit pick. But could make the style consistent by using 4 spaces?